src/main/java/uk/ac/ebi/subs/fileupload/util/FileStatus.java

```diff
@@ -4,5 +4,5 @@
  * An enumerated list of file statuses.
  */
 public enum FileStatus {
-    INITIALIZED, UPLOADING, UPLOADED
+    INITIALIZED, UPLOADING, UPLOADED, READY_TO_ARCHIVE
```

Comment:
As discussed, I think status names should describe the current status of the entity, READY_TO_ARCHIVE seems very forward looking. Please consider renaming this.

Reply:
Done